### PR TITLE
Transfer data related code to a separate package

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

**Review by roxblnfk — COMMENTED** [review]
Need to remove links to the `entity-reader.md` files in `./docs/*/README.md` files

**Review by samdark — APPROVED** [review]

File: docs/en/README.md [hunk 1]
```diff
@@ -9,5 +9,10 @@ This package provides a convenient way to integrate it with Yii framework.
...
 - [Installation and configuring](installation.md)
 - [Console commands](console-commands.md)
-- [EntityReader](entity-reader.md)
 - [Reading DB schema](reading-schema.md)
+
+# Other packages
+
+There is also an additional package providing Cycle ORM query adapter for [Yii Data](https://github.com/yiisoft/data) 
+data providers - [Yii Data Cycle](https://github.com/yiisoft/data-cycle). You can use it independently or in combination 
+with this package.
```
Comment (roxblnfk) [inline docs/en/README.md:18]: Shouldn't it be added in the ES docs?

**Review by vjik — APPROVED** [review]

File: docs/en/README.md (hunk 1 above)
Comment (arogachev) [reply docs/en/README.md:18]: Yes, but what about the translation?

Comment (arogachev) [reply docs/en/README.md]: #196

**Pull request merged**